EIPS/eip-xxxx.md

```diff
@@ -0,0 +1,131 @@
+---
+eip: xxxx
```

Comment:
```suggestion
eip: 7839
```
Assigning next sequential EIP/ERC/RIP number.

Please also update the filename.